css_tasks/task9/js/app.js

```diff
@@ -0,0 +1,7 @@
+import {DOMElements as elems} from './elements.js';
+import {handleTabs} from './tabs.js';
+const {tabs, texts} = elems;
+
+tabs.forEach((tab) => {
+  tab.addEventListener('click', (e) => handleTabs(e.target, texts));
```

Comment:
I've already send it to you, just in case
https://javascript.info/event-delegation
https://javascript.info/bubbling-and-capturing